hw 3/task 2.py

```diff
@@ -0,0 +1,23 @@
+# Строка для примера
+# это механизм позволяющий получить случайное слово или список слов по заданным параметрам пользователя
+
+def space_count(val, len_s):
+    return (len_s - len(val)) * ' '
+
+STEP = 1
+count = 1
+len_str = 0
+INDENT = 10
+
+user_str = input('Введите строку: ')
+ar = user_str.split()
+ar.sort()
+
+for el in ar:
+    x = len(el)
+    if len_str < x: len_str = x
```

Comment:
Длину самого длинного слова можно найти гораздо проще, используя встроенные агрегирующие функции. Например так
```suggestion
len_str = len(max(ar, key=len))
```